TRB_label_assist/SVM_decision_boundaries.ipynb

```diff
@@ -60,10 +62,12 @@
                 "uuids = [suburban_uuid, college_campus_uuid]\n",
                 "confirmed_trip_df_map = {}\n",
                 "labeled_trip_df_map = {}\n",
+                "ct_entry={}\n",
                 "expanded_trip_df_map = {}\n",
                 "for u in uuids:\n",
                 "    ts = esta.TimeSeries.get_time_series(u)\n",
-                "    ct_df = ts.get_data_df(\"analysis/confirmed_trip\")\n",
+                "    ct_entry[u]=eamtr._get_training_data(u,None)\n",
+                "    ct_df = ts.to_data_df(\"analysis/confirmed_trip\",itertools.chain(ct_entry[u]))\n",
```

Comment:
why do we need `itertools.chain` here? `find_entries` already returns a list of entries, IIRC
Please add code comment since the related blame does not have any explanation

Reply:
we do not. Following the code trail, I felt it would be needed, but the code runs fine without. REMOVED.